Lean4Axiomatic/Rational/Impl/Fraction/Reciprocation.lean

```diff
@@ -191,27 +191,40 @@ theorem div_eqv_fraction
   a//b                 ≃ _ := eqv_refl
 
 /--
-Every fraction can be expressed as a ratio of integers.
-
-This is a trivial theorem for fractions, but there may be other rational number
-representations for which the proof is much more difficult.
-
-**Property intuition**: Formal fractions satisfy this by definition.
-
-**Proof intuition**: Use the previous result that formal fractions are
-equivalent to division of integers, and that the denominator is nonzero because
-it's positive.
--/
-def as_ratio (p : Fraction ℤ) : AsRatio p := by
-  revert p; intro (a//b)
-  show AsRatio (a//b)
-  have : Integer.Nonzero b := Integer.nonzero_from_positive_inst
-  have : a//b ≃ a / b := eqv_symm div_eqv_fraction
-  exact AsRatio.intro a b ‹Integer.Nonzero b› this
+Every fraction satisfies the rational induction axiom, that is
```

Comment:
Great explanation! Could you wrap all variable references / code expressions in backticks please?